Gemfile.lock

```diff
@@ -495,6 +499,13 @@ GEM
     shellany (0.0.1)
     shoulda-matchers (3.0.0)
       activesupport (>= 4.0.0)
+    sidekiq (4.2.2)
```

Comment:
We don't want these to show up in the committed Gemfile.lock, right, since it won't be the default configuration?